### PR TITLE
Add Caps Lock → Meh (⌃⌥⇧) (Caps Lock if alone)

### PR DESCRIPTION
**Pull request by jchatard**

### CODE REVIEW

**Comment by MuhammedZakir** [comment]
Why do you call that key combo "Meh"? Just curious.

**Comment by jchatard** [comment]
I don't really know, it's from Oryx, the front end for QMK from ZSA: https://configure.zsa.io/ergodox-ez/layouts/default/latest/0/58/  Look at the right part of the split keyboard...

For tabs, yeah sure!

File: public/json/meh_capslock.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+	"title": "Caps Lock → Meh (⌃⌥⇧) (Caps Lock if alone)",
+	"rules": [
+		{
+			"description": "Caps Lock → Meh Key (⌃⌥⇧) (Caps Lock if alone)",
+			"manipulators": [
+				{
+					"from": {
+						"key_code": "caps_lock"
```
[the remark this answers is not in the document]
Comment (jchatard) [reply public/json/meh_capslock.json:9]: Ah ah I have no idea what you're talking about 🤣

[the remark this answers is not in the document]
Comment (MuhammedZakir) [reply public/json/meh_capslock.json]: Yeah, except you missed(?) `key_code`:
```json
"from": {
  "key_code": "caps_lock",
  "modifiers": {
    "optional": [ "caps_lock" ]
  }
}
```

[the remark this answers is not in the document]
Comment (jchatard) [reply public/json/meh_capslock.json]: Thank you!

**Pull request merged**